pkg/flare/archive.go

```diff
@@ -746,6 +769,41 @@ func zipHTTPCallContent(tempDir, hostname, filename, url string) error {
 	return err
 }
 
+func zipPerformanceProfile(tempDir, hostname, profileDir string) error {
+	srcHeapPath := filepath.Join(profileDir, HeapProfileName)
+	srcHeapProfile, err := os.OpenFile(srcHeapPath, os.O_RDWR|os.O_CREATE, os.ModePerm)
+	if err != nil {
+		return err
+	}
+
+	srcCPUPath := filepath.Join(profileDir, CPUProfileName)
+	srcCPUProfile, err := os.OpenFile(srcCPUPath, os.O_RDWR|os.O_CREATE, os.ModePerm)
+	if err != nil {
+		return err
+	}
+
+	dstHeapPath := filepath.Join(tempDir, hostname, HeapProfileName)
+	dstHeapProfile, err := os.OpenFile(dstHeapPath, os.O_RDWR|os.O_CREATE, os.ModePerm)
+	if err != nil {
+		return err
+	}
+
+	dstCPUPath := filepath.Join(tempDir, hostname, CPUProfileName)
+	dstCPUProfile, err := os.OpenFile(dstCPUPath, os.O_RDWR|os.O_CREATE, os.ModePerm)
+	if err != nil {
+		return err
+	}
+
+	_, err = io.Copy(dstHeapProfile, srcHeapProfile)
+	if err != nil {
+		return err
+	}
+
+	_, err = io.Copy(dstCPUProfile, srcCPUProfile)
+
+	return err
```

Comment:
There is a`CopyFile` function: https://github.com/DataDog/datadog-agent/blob/18d5c0811f2a9de88f2f48d77eaae19b8c04eea4/pkg/util/common.go#L30.

Reply:
Ah that's nice, didn't know that.